src/EditorFeatures/CSharpTest/Formatting/Indentation/SmartIndenterTests.cs

```diff
@@ -65,7 +65,7 @@ public void EndOfFileInactive()
             AssertSmartIndent(
                 code,
                 indentationLine: 4,
-                expectedIndentation: 4);
+                expectedIndentation: 0);
```

Comment:
this is a change in behavior.  however, i don't see anything about the previous behavior that seems desirable and warrants the necessary code to support this scenario.

Reply:
Agreed previous behavior seems...odd...